jenkins/jenkins-install.sh

```diff
@@ -57,18 +64,25 @@ else
 fi
 
 echo "Configuring the firewall to allow Jenkins..."
```

Comment:
Is this an issue? Why are you adding this? Where is the issue tracker where this is tracked and we can discuss why we would add or do this work?

Reply:
Seems to be conflating `firewalld` & `ufw`.
86.205% sure centOS/RH & UUbuntu/debian include each by default respectively.

& I believe most these files were the initial commits to master as a starting point to branch off of.